frontend/awx/access/organizations/OrganizationPage/OrganizationDetails.tsx

```diff
@@ -37,79 +37,61 @@ export function OrganizationDetails() {
   const { t } = useTranslation();
   const params = useParams<{ id: string }>();
   const { data: organization } = useGetItem<Organization>(awxAPI`/organizations/`, params.id);
-  const pageNavigate = usePageNavigate();
   const getPageUrl = useGetPageUrl();
 
+  const columns = useOrganizationsColumns();
   const galaxyCredentials = useGalaxyCredentials(params.id || '0');
   const instanceGroups = useInstanceGroups(params.id || '0');
 
+  const detailColumns = useMemo(
+    () => columns.filter((col) => col.id !== 'execution-environment'),
```

Comment:
max_hosts as well?

Reply:
When do we want to show max_hosts and when do we not?

Reply:
Based on the license type. That comes from the config API endpoint. ```'api/v2/config/'```

``` {license_info?.license_type !== 'open```